spec.html

```diff
@@ -438,11 +438,12 @@ <h1>%PrivateName% ( )</h1>
       </emu-clause>
 
       <emu-clause id="sec-private-name-object" aoid=PrivateNameObject>
-        <h1>PrivateNameObject ( _name_ )</h1>
+        <h1>PrivateNameObject ( _name_, _restricted_ )</h1>
```

Comment:
I wonder if it might be better to name this "privileged" instead of "restricted", so that behavior can default to the safe thing instead of to the unsafe thing?

Reply:
Are you suggesting a change in runtime semantics, or just an editorial change?

Reply:
An editorial change, to invert the default from "privileged" to "restricted".

Reply:
The design guideline used in ES5 when designing property descriptors:
> Names should say what is allowed rather than what is denied. APIs should follow the security best practice of "deny by default". 

Applying that guideline to this API the second parameter might be called readWriteAllowed and should default to false.

Reply:
I understand that it isn't a public API, but it is still an API used by spec. writers and readers.

I don't really care what name you chose as long as it describes what setting it to true enables.